src/components/Channel/Channel.tsx

```diff
@@ -833,7 +877,20 @@ export const ChannelWithContext = <
         ...client.user,
       },
       ...extraFields,
-    } as unknown) as MessageResponse<At, Ch, Co, Me, Re, Us>);
+    } as unknown) as MessageResponse<At, Ch, Co, Me, Re, Us>;
+
+    if (preview.quoted_message_id) {
+      const quotedMessage = messages.find(
+        (m) => m.id === preview.quoted_message_id,
+      );
+
+      preview.quoted_message = quotedMessage as Omit<
```

Comment:
@nhannah @virdesai Is typecasting the appropriate thing here?  Or any better way to do it?

Merging PR for now. But let me know if this can be changed!!

Reply:
Nope that's right